2023/did-wg.html

```diff
@@ -151,12 +150,10 @@ <h2>Motivation and Background</h2>
 level changes on the Recommendation.
         </p>
         <p>
-The Working Group will also seek consensus around DID Method specifications, and
-may produce related documents.
+The Working Group will also take over the work started by the Credentials Community Group around <a href="https://w3c-ccg.github.io/did-resolution/">DID Resolution</a>, to define common requirements, algorithms including their inputs and results, architectural options, and various considerations for the DID resolution and DID URL dereferencing processes.
         </p>
         <p>
-The Working Group will begin working toward a specification for DID Resolution
-and DID URL Dereferencing.
+It is expected that these two goals will be taken by distinct task forces in the WG.
```

Comment:
```suggestion
It is expected that these two goals will be taken by distinct task forces in the WG.

The Working Group MUST pursue consensus in any task pursued under the name of the working group--whether by Chairs, Editors, or other contributors--including any Notes for publication by the WG and any Charters for a future DID WG.
```

Reply:
Along with my proposed changes, I support this change and have budgeted my time and energy to continue to escalate objections against any further work planned by this WG that could _possibly_ abuse process in the ways that I have so painfully observed when its members are not properly constrained.

Reply:
I have always assumed that this proposed language applies anyway, but I guess it helps to emphasize it, so I'd support this too.

Reply:
> I have always assumed that this proposed language applies anyway

I would be willing to resubmit a pull request substantially similar to [DID Implementation Guide pull #36](https://github.com/w3c/did-imp-guide/pull/36) if this WG, under the current heightened scrutiny, agrees that pursuing consensus is required for Notes.

Reply:
Note that although the email summaries reference the original change proposed by @pchampin , which looks like: 

> It is expected that these two goals will be taken by distinct task forces in the WG.

the current text under discussion is: 

> The Working Group MUST pursue consensus in any task pursued under the name of the working group--whether by Chairs, Editors, or other contributors--including any Notes for publication by the WG and any Charters for a future DID WG.

So if you are only reading along in email then you will be misled.

Reply:
@jandrieu just to be clear about the changes you suggest above :
by "pursue consensus", do you mean to rule out completely recourse to [Section 5.2.2. Managing Dissent](https://www.w3.org/2023/Process-20231103/#managing-dissent) and [Section 5.2.3. Deciding by Vote](https://www.w3.org/2023/Process-20231103/#Votes) of the process?

Reply:
I do not. I mean to make sure that staff & chairs don't assert that actions taken in the name of the group do not require the seeking of consensus.